cardscan-base/src/main/java/com/getbouncer/cardscan/base/UXModelResult.java

```diff
@@ -0,0 +1,56 @@
+package com.getbouncer.cardscan.base;
+
+public class UXModelResult {
+
+    public final float noCardScore;
+    public final float panSideScore;
+    public final float noPanSideScore;
+    private float[] modelOutput;
+    private float maxScore;
+    private UXModelEnum uxModelEnum;
+
+    public UXModelResult(float[] modelOutput) {
+        noCardScore = modelOutput[0];
+        panSideScore = modelOutput[1];
+        noPanSideScore = modelOutput[2];
+        this.modelOutput = modelOutput;
+
+        this.calculateResult();
+    }
+
+    public enum UXModelEnum {
+        NO_PAN_SIDE,
+        NO_CARD,
+        PAN_SIDE
+    }
```

Comment:
this is fine for this codebase. Normally, because we're developing an SDK that may expand in the future, we would use integers instead of an enum to describe state. This will prevent compilation problems if we add more states in the future.